javalib/src/main/scala/java/util/EnumSet.scala

```diff
@@ -2,27 +2,113 @@ package java.util
 
 import java.lang.Enum
 
-final class EnumSet[E <: Enum[E]] private (values: Array[E])
+final class EnumSet[E <: Enum[E]] private (values: Set[E])
```

Comment:
this could still be an array where the index corresponds to ordinal, and the max ordinal + 1 is the size of the array? (never shrink, only grow by powers of 2?)

Reply:
Sure, it could have been, but it's not going to be frequently used, and that way we get the Set ops for free. At some point it could be rewriten to be more optimal, but for current usages its good enough, especially since we cannot support the relection based semantics to populate it with all values